dotnet/samples/KernelSyntaxExamples/Example49_LogitBias.cs

```diff
@@ -44,14 +44,14 @@ public static async Task RunAsync()
         await MessageOutputAsync(chatHistory);
 
         var replyMessage = await chatCompletionService.GetChatMessageContentAsync(chatHistory, settings);
-        chatHistory.AddAssistantMessage(replyMessage.Content);
+        chatHistory.AddAssistantMessage(replyMessage.Content!);
```

Comment:
Why are we certain (using the `!` operator) that the content is not null here?

Reply:
That's a good question. Currently, we operate with `ChatMessageContent` as model for request and response at the same time. In case of request, property `Content` can be used for plain text or property `Items` for multi-modal request. While for response, only `Content` property is used. That's why `Content` property is nullable, because for multi-modal request it's not required.. I think we can either use different classes for chat message request and response scenario or remove `Content` property completely and use only `Items` in request and response. In this case, we will avoid null checks. Although, with `Items`, users will have to do `Items[0]` to get response every time, so we will have to provide additional methods for better usability.

In this specific example, we know that `Content` is not null, because we set a response from AI into this property, and in case of some issue we just throw an exception.